### PR TITLE
latest thread query issue with order fixed

### PR DESCRIPTION
**Pull request by rtamizh**
Latest threads on user profiles is in wrong sequence #256 fix

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -154,7 +154,7 @@ public function threads()
      */
     public function latestThreads(int $amount = 3)
     {
-        return $this->threadsRelation->take($amount);
+        return $this->threadsRelation()->limit($amount)->get();
```
Comment (driesvints) [inline app/User.php]: I think this needs to be the following, no?

```php
return $this->threadsRelation()->latest()->take($amount);
```

Comment (rtamizh) [inline app/User.php]: I did a mistake. But still we have to give like this

`return $this->threadsRelation()->latest()->limit($amount)->get();`

Comment (driesvints) [inline app/User.php]: Ok perfect, update your PR and I'll merge it :)

**Comment by driesvints** [comment]
Thanks! :)

**Pull request merged**